database/factories/Multiplayer/GameFactory.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+use Carbon\Carbon;
+
+$factory->define(App\Models\Multiplayer\Game::class, function (Faker\Generator $faker) {
+    $beatmap = App\Models\Beatmap::inRandomOrder()->first();
+
+    return [
+        'beatmap_id' => $beatmap->beatmap_id,
+        'start_time' => Carbon::now()->subSeconds($beatmap->total_length),
+        'play_mode' => $faker->numberBetween(0, 3),
```

Comment:
I'd just use the beatmap's playmode to ensure we're using a valid playmode for the beatmap